news/2024/2024-09-02-the-followpoint-kaito-thighs-the-picasso-of-osu!.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: post
+title: "The Followpoint: KAITO thighs, the Picasso of osu!"
+date: 2024-09-02 12:00:00 +0000
+series: The Followpoint
+---
+
+The Followpoint has came back with another interview! No, we are not interviewing about someones thighs today, just art.
```

Comment:
if we aren't using Dreamxiety's proper username in the title then i suggest at least mentioning it in the intro text. could have a `(better known as Dreamxiety)` somewhere here.

Reply:
reason why i'd want it on line 8 so it's immediately cleared up in newspost previews (see image), would suggest rewriting with that in mind.

![](https://nats.are-la.me/7ixqmN9.png)

Reply:
Could it maybe something like `The Followpoint has came back, this time with KAITO thighs (better known as Dreamxiety). No, we are not interviewing about someone's thighs today, just art.` or is that too long

Reply:
Can we not name the post "thighs"? It's a bad look.

Please use "Dreamxiety".

Reply:
will do, thanks for your response